CHANGELOG.md

```diff
@@ -9,6 +30,35 @@
 * **[Fix]** Downgrade the platforms verification error to a warning.
 * **[Fix]** Update vulnerable `Microsoft.NETCore.UniversalWindowsPlatform` dependency from version `5.2.2` to `5.2.6`.
 
+#### ANDROID
+
+* **[Fix]** Fix a bug where crash data file could leak when the database is full.
+
+#### IOS
+
+* **[Feature]** Add preview support for arm64e CPU architecture by using PLCrashReporter 1.2.3-rc1. PLCrashReporter 1.2.3-rc1 is a fork of the official repository and can be found at https://github.com/Microsoft/PLCrashReporter. It is based on version It is maintained by the App Center team and based on PLCrashReporter 1.2.1 (commit fda23306) with additional fixes and changes.
+
+### AppCenterAnalytics
+
+#### IOS
+
+* **[Feature]** Add preview support for arm64e CPU architecture.
+
+### AppCenterDistribute
+
+#### IOS
+
+* **[Feature]** Add preview support for arm64e CPU architecture.
+
+### AppCenterPush
+
+#### IOS
+
+* **[Feature]**  Add preview support for arm64e CPU architecture.
+
+#### ANDROID
+
+ * **[Fix]** Fix push foreground listener after re-enabling push service.
```

Comment:
On Xamarin this bug fix needs to be documented differently:

**[Fix]** Fix `PushNotificationReceived` event for pushes received in foreground after re-enabling the push service.